CHANGELOG.rst

```diff
@@ -4,6 +4,32 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v10.0.0
+=======
+
```

Comment:
Why I don't see https://github.com/ansible-collections/community.aws/pull/2303/files? Did I miss it? I guess you need to a changelog_fragment to take in consideration this change.

Reply:
I added info about the change from #2303 in the "breaking changes" section [here](https://github.com/ansible-collections/community.aws/pull/2307/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR24).

Reply:
Just a note - it looks like #2303 should have included a breaking change changelog fragment. In general, we aim to skip changelogs only for trivial or test-related changes.”

Reply:
Yep, that makes sense. Sorry for the oversight!